qiskit_experiments/library/characterization/fine_frequency.py

```diff
@@ -0,0 +1,130 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Fine frequency characterization experiment."""
+
+from typing import List, Optional
+import numpy as np
+
+from qiskit import QuantumCircuit, schedule, transpile
+from qiskit.providers.backend import Backend
+
+from qiskit_experiments.framework import BaseExperiment, Options
+from qiskit_experiments.library.characterization.analysis import FineAmplitudeAnalysis
+
+
+class FineFrequency(BaseExperiment):
+    r"""An experiment to make a fine measurement of the qubit frequency.
+
+    The fine frequency characterization experiment measures the qubit frequency by moving
```

Comment:
Please use sections. This should be overview. Note that now you need to manually write `analysis_ref` section in the base class since class attribute for analysis is removed. See other classes for details.

Reply:
Done here: fa9247e